bin/_mocha

```diff
@@ -63,8 +63,8 @@ program
   .option('-c, --colors', 'force enabling of colors')
   .option('-C, --no-colors', 'force disabling of colors')
   .option('-G, --growl', 'enable growl notification support')
-  .option('-O, --reporter-options <k=v,k2=v2,...>', 'reporter-specific options')
-  .option('-R, --reporter <name>', 'specify the reporter to use', 'spec')
+  .option('-O, --reporter-options <k=v,k2=v2,...>', 'reporter-specific options, use name1:{k=v,k2=v2,...},name2:{...} for multiple reporters')
```

Comment:
Sure, want me to add a dependency to subarg and use that instead? IIRC, This syntax was suggested on the previous PR, but I have no issues using a different one. 

However, I do like the fact  that is the same syntax for the reporter options when using a single reporter, so you can just copy and paste them without the need of reformatting. e.g.: 

`mocha --reporter foo --reporter-options x=y,w=z` 
is equivalent to: 
`mocha --reporter foo --reporter-options foo:{x=y,w=z}`